src/ontology/mondo-edit.obo

```diff
@@ -20078,34 +20078,25 @@ is_a: MONDO:0001370 {source="DOID:115"} ! pericardial effusion
 [Term]
 id: MONDO:0001298
 name: congenital mitral valve insufficiency
-def: "Dysfunction of the mitral valve characterized by incomplete valve closure." [NCIT:P378]
+def: "Mitral valve insufficiency that is present at birth." [NCIT:C197881]
 synonym: "congenital insufficiency of mitral valve" EXACT []
-synonym: "congenital mitral insufficiency" EXACT [DOID:11502]
+synonym: "congenital mitral insufficiency" EXACT [DOID:11502, NCIT:C197881]
 synonym: "congenital mitral regurgitation" EXACT [DOID:11502]
```

Comment:
@sabrinatoro if I remove DOID from `synonym: "congenital mitral insufficiency" EXACT [DOID:11502, NCIT:C197881]` on [Line 20083](https://github.com/monarch-initiative/mondo/pull/8147/files#diff-889e0a9ba62d62cc6ee7c5c9bead0075444d506253590ef0dcf4786904ac6768R20083) then shouldn't the DOID also be removed from this line:
`synonym: "congenital mitral regurgitation" EXACT [DOID:11502]`

Reply:
yes, it should be. 
All the exact synonyms from DOID:11502 should be removed. 
(since DOID:11502 is not an equivalent x-ref, the synonyms from this x-ref should not be exact).

Reply:
remove the synonym or make it a related synonym?

Reply:
remove

Reply:
ok, i will remove it, but also think it would be useful for someone trying to find this concept to leave the synonym on the term